chat-plugins/faq.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const ROOMFAQ_FILE = path.resolve(__dirname, '../config/chat-plugins/faqs.json');
+
+let roomFaqs = {};
+try {
+	roomFaqs = require(ROOMFAQ_FILE);
+} catch (e) {} // file doesn't exist or contains invalid JSON
```

Comment:
Apparently I just didn't notice that you took it out without doing it properly. I actually told you how to do it properly in the PR comments: `e indexof SyntaxError`. :|